saleor/app/models.py

```diff
@@ -168,6 +170,7 @@ class AppExtension(models.Model):
         blank=True,
         help_text="Specific permissions for this app extension.",
     )
+    # TODO Remove in 3.23 or earlier - options resolver can extract from setting
```

Comment:
Hhm, now I am confused 😅  3.23 is `main`. 3.22 is ealier. In that case why do we make TODOs, instead of making the clean of this change?